eng/versioning/version_client.txt

```diff
@@ -143,51 +143,51 @@ com.azure:azure-messaging-servicebus;7.14.3;7.15.0-beta.4
 com.azure:azure-messaging-servicebus-stress;1.0.0-beta.1;1.0.0-beta.1
 com.azure:azure-messaging-servicebus-track1-perf;1.0.0-beta.1;1.0.0-beta.1
 com.azure:azure-messaging-servicebus-track2-perf;1.0.0-beta.1;1.0.0-beta.1
-com.azure:azure-messaging-webpubsub;1.2.6;1.3.0-beta.1
+com.azure:azure-messaging-webpubsub;1.2.7;1.3.0-beta.1
 com.azure:azure-messaging-webpubsub-client;1.0.0-beta.1;1.0.0-beta.2
-com.azure:azure-mixedreality-authentication;1.2.15;1.3.0-beta.1
-com.azure:azure-mixedreality-remoterendering;1.1.20;1.2.0-beta.1
+com.azure:azure-mixedreality-authentication;1.2.16;1.3.0-beta.1
+com.azure:azure-mixedreality-remoterendering;1.1.21;1.2.0-beta.1
 com.azure:azure-monitor-opentelemetry-exporter;1.0.0-beta.11;1.0.0-beta.12
-com.azure:azure-monitor-ingestion;1.0.5;1.1.0-beta.1
+com.azure:azure-monitor-ingestion;1.0.6;1.1.0-beta.1
 com.azure:azure-monitor-ingestion-perf;1.0.0-beta.1;1.0.0-beta.1
-com.azure:azure-monitor-query;1.2.2;1.3.0-beta.2
+com.azure:azure-monitor-query;1.2.4;1.3.0-beta.2
 com.azure:azure-monitor-query-perf;1.0.0-beta.1;1.0.0-beta.1
 com.azure:azure-perf-test-parent;1.0.0-beta.1;1.0.0-beta.1
 com.azure:azure-quantum-jobs;1.0.0-beta.1;1.0.0-beta.2
-com.azure:azure-search-documents;11.5.9;11.6.0-beta.9
+com.azure:azure-search-documents;11.5.10;11.6.0-beta.9
 com.azure:azure-search-perf;1.0.0-beta.1;1.0.0-beta.1
-com.azure:azure-security-attestation;1.1.15;1.2.0-beta.1
-com.azure:azure-security-confidentialledger;1.0.11;1.1.0-beta.1
-com.azure:azure-security-keyvault-administration;4.3.4;4.4.0-beta.1
-com.azure:azure-security-keyvault-certificates;4.5.4;4.6.0-beta.1
+com.azure:azure-security-attestation;1.1.16;1.2.0-beta.1
+com.azure:azure-security-confidentialledger;1.0.12;1.1.0-beta.1
+com.azure:azure-security-keyvault-administration;4.3.5;4.4.0-beta.1
+com.azure:azure-security-keyvault-certificates;4.5.5;4.6.0-beta.1
 com.azure:azure-security-keyvault-jca;2.7.1;2.8.0-beta.1
 com.azure:azure-security-test-keyvault-jca;1.0.0;1.0.0
-com.azure:azure-security-keyvault-keys;4.6.4;4.7.0-beta.1
-com.azure:azure-security-keyvault-secrets;4.6.4;4.7.0-beta.1
+com.azure:azure-security-keyvault-keys;4.6.5;4.7.0-beta.1
+com.azure:azure-security-keyvault-secrets;4.6.5;4.7.0-beta.1
 com.azure:azure-security-keyvault-perf;1.0.0-beta.1;1.0.0-beta.1
 com.azure:azure-sdk-template;1.1.1234;1.2.2-beta.1
 com.azure:azure-sdk-template-two;1.0.0-beta.1;1.0.0-beta.1
 com.azure:azure-sdk-template-three;1.0.0-beta.1;1.0.0-beta.1
-com.azure:azure-spring-data-cosmos;3.37.0;3.38.0-beta.1
-com.azure:azure-storage-blob;12.23.0;12.24.0-beta.2
-com.azure:azure-storage-blob-batch;12.19.0;12.20.0-beta.2
+com.azure:azure-spring-data-cosmos;5.4.0;3.38.0-beta.1
```

Comment:
latest dependency is 5.4.0 which is correct.
https://mvnrepository.com/artifact/com.azure/azure-spring-data-cosmos

Reply:
Yeah, I'm confused about this one because they are maintaining two different GA versions: `3.37.0` and `5.4.0`. I don't think it will affect the BOM release process, but we should ask the Spring team why this is the case.

Reply:
Will revert this change.